### PR TITLE
Update PARTICIPANTS

### PR DESCRIPTION
**Pull request by qservando**
#1 referenced!

### CODE REVIEW

File: PARTICIPANTS/qservando.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name: qservando
```
Comment (MJ10) [inline PARTICIPANTS/qservando.txt]: Move the file to `Participants` directory

Comment (qservando) [reply PARTICIPANTS/qservando.txt]: Thank you, I will go ahead and do that.

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by qservando** [comment]
Updated to correct directory and add a note. Thank you.

**Comment by qservando** [comment]
PR can be closed.

**Comment by salman-bhai** [comment]
spam

**Pull request closed without merging**